src/cli2/common.test.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import {loadJson, loadJsonWithDefault} from "./common";
+import tmp from "tmp";
+import fs from "fs-extra";
+import * as C from "../util/combo";
+
+describe("cli2/common", () => {
+  function tmpWithContents(contents: mixed) {
+    const name = tmp.tmpNameSync();
+    fs.writeFileSync(name, JSON.stringify(contents));
+    return name;
+  }
+  describe("loadJson / loadJsonWithDefault", () => {
+    const fooParser = C.object({foo: C.number});
+    const badPath = "not-a-real-path";
```

Comment:
This file *could* exist. Could you create a temp directory and put
`goodPath` and `badPath` in there so that the test is actually hermetic?

Reply:
Did it for badPath